src/test/scala/org/broadinstitute/dsde/firecloud/webservice/TrialApiServiceSpec.scala

```diff
@@ -141,29 +147,73 @@ final class TrialApiServiceSpec extends BaseServiceSpec with UserApiService with
 
     // TODO: Test updates of users who aren't registered (i.e. no RegistrationInfo in Sam)
 
-    // TODO: Test invalid trial status transition request (e.g. Terminated -> Enabled)
+    "Thurloe errors" - {
+      "preventing us from getting user trial status should return a server error to the user" in {
+        Post(enablePath, dummy1UserEmails) ~> dummyUserIdHeaders(dummy1User) ~> trialApiServiceRoutes ~> check {
+          assertResult(InternalServerError, response.entity.asString) { status }
+        }
+      }
```

Comment:
This doesn't test server errors *getting* the trial status - there's nothing in `TrialApiServiceSpecThurloeDAO` that fails to get a status - it only fails saving the status. i.e. `TrialApiServiceSpecThurloeDAO` only overrides `saveTrialStatus`; it doesn't override `getTrialStatus`.